styles/notion.css

```diff
@@ -308,7 +338,7 @@
   padding: 0;
   margin: 0;
   border-radius: 0;
-  background: none !important;
+  background-image: none !important;
```

Comment:
this is intentional for my site. i don't really like how the highlight colors look in dark mode, so i decided to just disable them entirely.

happy to take suggestions on how to make things look better tho :)

Reply:
Yeah, I thought it might be the reason. Here is a diff between how they lock in dark/light modes if it helps.

![image](https://user-images.githubusercontent.com/3807458/161917396-ad683ede-fa4c-411f-a537-f00796217f13.png)
![image](https://user-images.githubusercontent.com/3807458/161917442-e782bef2-4095-4e12-8327-18bb675399a2.png)